### PR TITLE
docs(experimenter): July's month newsletter

### PR DESCRIPTION
**Pull request by yashika-khurana**
## Description
- July's month newsletter

Screenshot:
image

### CODE REVIEW

File: docs/whats-news/2022/2022-07.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+### Feedback
+
+Now a feedback [link](https://mozilla-hub.atlassian.net/secure/CreateIssueDetails!init.jspa?pid=10203&issuetype=10097) is also available on the landing page. Don’t wait to give us any feedback!
+
+### Feature enabled
+
+- #### Desktop
+
+  Firefox 104 is removing support for Feature.enabled, so recipes targeting it should not use the Feature.enabled flag. As a result, Experiments for Firefox Desktop 104.And above requires you to have the feature enabled and need to specify the feature value too.
```
Comment (jaredlockhart) [inline docs/whats-news/2022/2022-07.md:35]:
```
Firefox Desktop 104 is removing support for the feature enabled toggle, so experiments Firefox Desktop 104 and above will require feature enabled to be on for every branch.
```

File: docs/whats-news/2022/2022-07.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+- #### Mobile
+  The feature enabled flag was never implemented on mobile, so allowing it in the UI creates the false impression that it has some meaning for mobile experiments. So we have removed the feature-enabled flag for mobile experiments and forced them to specify a feature value for every branch, which may explicitly be an empty object.
```
Comment (jaredlockhart) [inline docs/whats-news/2022/2022-07.md:38]:
```
The feature enabled flag has been disabled entirely for mobile experiments.  Every branch of a mobile experiment must specify a feature value, which may explicitly be an empty object "{}".
```

File: docs/whats-news/2022/2022-07.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+### Sticky Enrollemt
+
+- When creating a new experiment, users can now opt for sticky enrollment (clients will not be unenrolled, even if the targeting becomes false, until the experiment ends). This information is now visible on the experiment’s "Details" page so that the user can review if they have opted for sticky enrollment.
+
+- In addition to that, a new warning has been added when creating a new experiment. If an experiment is using a targeting configuration that requires sticky enrollment, sticky enrollment would be auto selected to support that targeting expression.
+
+### Normandy
+
+Normandy will continue to be supported until Nimbus can take over all of its capabilities. So in the meanwhile, we have restored the create button in legacy.
```
Comment (jaredlockhart) [inline docs/whats-news/2022/2022-07.md:25]:
```
 Normandy will continue to be supported until Nimbus can take over for its responsibilities. So in the meanwhile, we have restored the create button in the old Experimenter interface.
```

**Review by jaredlockhart — CHANGES_REQUESTED** [review]
Couple small changes but looks great thanks @yashika-khurana !

**Review by shell1 — APPROVED** [review]
this looks good to me.  Thanks Yashika!   I'm approving

**Review by jaredlockhart — APPROVED** [review]
Thanks @yashika-khurana !

**Pull request merged**